app/views/deploys/_checklist.html.erb

```diff
@@ -6,8 +6,8 @@
   <ul class="deploy-checklist">
     <%- stack.checklist.each do |check| -%>
       <li class="deploy-checklist__item">
+        <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
         <label class="deploy-checklist__item__label">
-          <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
```

Comment:
yeah he asked me to take care of it. @vernalkick can you add the proper classes such that the "Create Deploy" button will be disabled when it has a `disabled` class?